parfive/config.py

```diff
@@ -143,6 +143,14 @@ class SessionConfig:
     """
     env: EnvConfig = field(default_factory=EnvConfig)
 
+    done_callbacks: Iterable[Callable] = tuple()
```

Comment:
```suggestion
    done_callbacks: Iterable[Callable[[str, str, Optional[Exception], None] = tuple()
```

Reply:
It only takes 3 arguments. I'll change that, thx :)